sceneManager.lua

```diff
@@ -1,6 +1,9 @@
 require "scene"
 require "scenes/test"
 
+require "spell"
+require "spells/fireBall"
```

Comment:
You can just use `fireball`, it's 1 word. Make sure you also rename the files from `fireBall` to `fireball`